handlers/tokenutil_test.go

```diff
@@ -0,0 +1,34 @@
+package handlers_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/atmiguel/cerealnotes/handlers"
+	"github.com/atmiguel/cerealnotes/models"
+	"github.com/atmiguel/cerealnotes/test_util"
+)
+
+func TestToken(t *testing.T) {
+	env := &handlers.Environment{nil, []byte("TheWorld")}
+
+	var num models.UserId = 32
+	bob, err := handlers.CreateTokenAsString(env, num, 1)
+	if err != nil {
+		panic(err)
```

Comment:
`panic` will stop the entire test suite from running. Use [t.Fatal](https://golang.org/pkg/testing/#T.Fatal) or t.Fatalf